keyboards/nightly_boards/alter/rev1/readme.md

```diff
@@ -0,0 +1,13 @@
+
+![Alter](imgur.com image replace me!)
```

Comment:
Image should be added or line removed

Reply:
https://github.com/qmk/qmk_firmware/pull/9579/commits/547f925c236ac7b2f0d39af76c48bff5df0194f6